distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -433,7 +434,8 @@ private static Integer findPortNumber(Integer portToTry) {
                 // increase timeout for CI environment
                 systemTimeout(TimeUnit.MINUTES.toMillis(10)),
                 when(Boolean.getBoolean("keepRuntimeFolder")).useOptions(keepRuntimeFolder()),
-                cleanCaches(true));
+                cleanCaches(true),
+                configureConsole().ignoreLocalConsole());
```

Comment:
This was from an example I found when trying to figure out other issues. I retested without it and it is not needed. Removed the ignoreLocalConsole